pallets/crowdloans/src/lib.rs

```diff
@@ -153,44 +153,38 @@ pub mod pallet {
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
         /// New vault was created
-        /// [para_id, ctoken_id]
-        VaultCreated(ParaId, AssetIdOf<T>),
+        /// [para_id, vault_id, ctoken_id, phase, contribution_strategy, cap, end_block, trie_index]
+        VaultCreated(
```

Comment:
crowdloans-subquery is based on event to record Vaults, we need more detailed vault information there.